cspell/.cspell.json

```diff
@@ -22,6 +22,8 @@
         "reports*/**"
     ],
     "words": [
+        "KHTML",
+        "etag"
```

Comment:
```suggestion
        "etag",
```

Reply:
Ah, sorry :pleading_face:

Reply:
I'll update the script to check this error.